app/pkg/protocol/state_proposal.go

```diff
@@ -5,18 +5,18 @@ import (
 	"errors"
 
 	"github.com/shopspring/decimal"
-	"github.com/statechannels/go-nitro/channel/state"
+	st "github.com/statechannels/go-nitro/channel/state"
 	"github.com/statechannels/go-nitro/types"
 )
 
 // StateProposal represents information about proposed state.
 type StateProposal struct {
-	state          *state.State
```

Comment:
state here is heavy one 129 byte(so lets use a pointer to state)

liabilityState is struct with map (delete pointer)